cg/meta/workflow/balsamic.py

```diff
@@ -232,6 +240,45 @@ def get_verified_normal_path(sample_data: dict) -> Optional[str]:
             return None
         return normal_paths[0]
 
+    def get_latest_raw_file_data(self, case_id: str, tags: list) -> Union[list, dict]:
```

Comment:
why return a list or a dict? Can we decide on one of them?

Reply:
@patrikgrenfeldt

The reason is that the BALSAMICs `config.json` is parsed as a dic, while `<case>_metrics_deliverables.yaml` is a list.

Reply:
would be nice for the functionality using this data to have only one format to rely on

Reply:
@patrikgrenfeldt 

Maybe a workaround to this could be making it even more general – following the MIP implementation (returning `Any`):

https://github.com/Clinical-Genomics/cg/blob/783143649909311ed2f7ffd6836b0b48c0156269/cg/meta/workflow/mip.py#L176

Updated in: [7831436](https://github.com/Clinical-Genomics/cg/pull/1446/commits/783143649909311ed2f7ffd6836b0b48c0156269)

Reply:
In that case I'd prefer `Union[list, dict]:` since it is clear that the method behaves a bit "weird"